src/fat.cpp

```diff
@@ -7,39 +7,16 @@
 // identity
 #include "orc/fat.hpp"
 
-// application
-#include "orc/mach_types.hpp"
+// mach-o
+#include <mach-o/loader.h>
```

Comment:
I finally came to my senses and am using the Mach-O types provided by the system instead of recreating them.